src/index.ts

```diff
@@ -1,42 +1,50 @@
 import scrypt = require('scrypt')
 
 import * as constants from './utils/constants'
-import { generateSeed } from './utils/helpers'
+import { generateSeed, toNetworkByte, computeTemplate } from './utils/helpers'
 import { templates, templateChars } from './utils/templates'
 
-export const generateKey = (name: string, password: string, namespace?: string, version?: string): Buffer => {
-  if (!namespace) {
+export const generateKey = (name: string, password: string, version: number = constants.MP_ALGORITHM_VERSION,
+  namespace?: string): Buffer => {
```

Comment:
Multiline function invocation should have an item on each line, i.e.

```typescript
export const generateKey = (
  name: string,
  password: string,
  version: number = constants.MP_ALGORITHM_VERSION,
  namespace?: string
): Buffer => {
  // Function content
}
```

Also, why use `if (typeof namespace === 'undefined')` below instead of a default arg here, in the same manner that `version` is defined?